src/components/common/EditButton.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button class="btn btn-outline-primary me-4 rounded-pill py-2 px-5">
-    Edit
+  <button class="btn btn-outline-primary me-4 rounded-pill py-2 px-3 px-md-5">
+    <slot>Edit</slot>
```

Comment:
I guess there is need to have more generic name for this component and have the button text as required parametr sice an edit button which saves or adds something doesn't make sense

Reply:
I mean slot is fine, but when the name is changed "Edit" as default value won't make sense as well